helix-term/src/ui/picker.rs

```diff
@@ -159,16 +159,32 @@ impl<T: 'static> Component for FilePicker<T> {
         // |picker   | |         |
         // |         | |         |
         // +---------+ +---------+
-        let render_preview = area.width > MIN_SCREEN_WIDTH_FOR_PREVIEW;
-        let area = inner_rect(area);
+        let area = inner_rect(
+            area,
+            cx.editor.config.file_picker.width_factor,
+            cx.editor.config.file_picker.height_factor,
+        );
+        let render_preview = area.width
+            > cx.editor.config.file_picker.preview_min_width
+                + cx.editor.config.file_picker.file_min_width;
         // -- Render the frame:
         // clear area
         let background = cx.editor.theme.get("ui.background");
         let text = cx.editor.theme.get("ui.text");
         surface.clear_with(area, background);
 
         let picker_width = if render_preview {
-            area.width / 2
+            let preview_width = (area.width as f32
+                * cx.editor.config.file_picker.preview_width_factor)
+                .round() as u16;
```

Comment:
Shouldn't we do ceil instead of round? Not sure.

Reply:
`ceil` enlages the preview width, `floor` enlarges filenames. I can't find reasons to decide for one or the other, then `round`.
But there may be boundary effects... I don't know...